core-tests/src/test/scala/flow/FlowBuilderTest.scala

```diff
@@ -17,11 +16,11 @@ class FlowBuilderTest extends TestKit(ActorSystem("FlowBuilder")) with FunSpecLi
   implicit val ec = system.dispatcher
 
   def runFlow(initialValues: Int*)(flowBuilder: FlowBuilder[Int, Int, Int]): Future[Seq[FlowResult[Int, Int]]] =
-    Source(Stream(initialValues: _*))
+    Source(LazyList(initialValues: _*))
```

Comment:
P sure the change from `Stream` to `LazyList` is a no-brainer here, but I would like @desmondyeung to bless it, so we don't get any nasty surprises.

Reply:
I probably should have mentioned it somewhere, but according to the official docs, it should be a 1:1 replacement https://docs.scala-lang.org/overviews/core/collections-migration-213.html#what-are-the-breaking-changes

Reply:
Only note here is whether or not we relied on the fact that Stream's head is not lazy, whereas LazyList is lazy both on head and tail. In 99% of cases it won't matter.